src/test/java/net/minecraftforge/debug/entity/living/ExtendedSpecialSpawnTest.java

```diff
@@ -1,3 +1,21 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016-2018.
```

Comment:
should be 2016-2019

Reply:
Okay, done. The Information should probably be updated in license-header.txt, too.